src/rust/engine/process_execution/src/lib.rs

```diff
@@ -213,6 +213,21 @@ pub struct ExecuteProcessRequest {
   pub is_nailgunnable: bool,
 }
 
+impl ExecuteProcessRequest {
+  fn user_facing_name(&self) -> String {
+    let mut buf = String::new();
+    buf.push_str("Exec: [");
```

Comment:
Small nit: the `[` at the beginning and the end make it hard to copy-paste the command in case I want to try it again on my own.
I know it's not the intended purpose right now (you'd have to be very quick to copy anything from a swimlane), but it might be later.

Reply:
Like I mentioned earlier I don't think it makes sense to optimize an EPR's user-facing name for copy-pasting, but I think I'll get rid of the `[]` anyway just to reduce the amount of less useful text on a line.